docs/InstallGuide.md

```diff
@@ -92,18 +81,59 @@ Modify the following section in /etc/openvnet/webapi.conf on the webapi node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
+        public ""
         port 9101
       }
     }
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the webapi process is run on the same node as vnmgr or vna, specify a different value from vnmgr and vna.
+- **host** : The private IP address of the webapi node.
```

Comment:
What is `private` in this context? There is no clear definition of what we mean by private and public IP address.